### PR TITLE
fix: fixed the snake case in users.json to camel case,

### PR DESCRIPTION
**Pull request by minjnlgc**
* fixed the snake case in users.json to camel case
* updated `decode_users` for reading assigned GP/patients attributes

### CODE REVIEW

File: breeze/models/appointment_entry.py [hunk 1]
```diff
@@ -89,11 +89,11 @@ def to_dict(self):
             dict: Dictionary representation of the appointment entry.
         """
         return {
-            "appointment_id": self.appointment_id,
+            "appointmentId": self.appointment_id,
```
Comment (abbieDuan87) [inline breeze/models/appointment_entry.py:92]: I think id are usually ID instead of Id?

File: breeze/utils/data_utils.py [hunk 2]
```diff
@@ -144,9 +146,9 @@ def create_appointments_from_data(appointments_data):
         time = app.get("time")
         is_cancel = app.get("isCancel", False)
         status = app.get("status", None)
-        mhwp_username = app.get("mhwp_username", None)
-        patient_username = app.get("patient_username", None)
-        appointment_id = app.get("appointment_id", None)
+        mhwp_username = app.get("mhwpUsername", None)
+        patient_username = app.get("patientUsername", None)
+        appointment_id = app.get("appointmentId", None)
```
Comment (abbieDuan87) [inline breeze/utils/data_utils.py:151]: if changed Id,  here needs to changed as well

File: breeze/models/mhwp.py [hunk 3]
```diff
@@ -32,7 +32,9 @@ def __init__(
             is_disabled=is_disabled,
         )
         self.__appointments = appointments if appointments is not None else []
-        self.__assigned_patients = assigned_patients if assigned_patients is not None else []
+        self.__assigned_patients = (
+            assigned_patients if assigned_patients is not None else []
```
Comment (abbieDuan87) [inline breeze/models/mhwp.py:36]: just out of curiosity, when will be the case that assigened_patients will be a tuple of list? 
if assigned_patients is always to be a list, then we can enforce it to be a list type like:
`self.__assigned_patients = list(assigned_patients) if assigned_patients is not None else []`

File: breeze/models/appointment_entry.py (hunk 1 above)
Comment (abbieDuan87) [reply breeze/models/appointment_entry.py:92]: sure lets got with Id

**Review by abbieDuan87 — APPROVED** [review]

**Review by rheahemmadi — APPROVED** [review]

**Pull request merged**